src/aleph_client/commands/domain.py

```diff
@@ -0,0 +1,313 @@
+        max_retries = 5
+        while dns_rules:
+            rule = dns_rules[0]
+            """Get rules check status"""
+            checks = await check_domain_records(fqdn, target, owner)
+            completed_rules = []
+            for index, rule in enumerate(dns_rules):
+                if checks[rule.name] is True:
+                    """Pass configured rules"""
+                    completed_rules.append(rule)
+                    console.log(f"record: {index} [bold green] OK")
+
+            for _rule in completed_rules:
+                dns_rules.remove(_rule)
+                completed_rules = []
+
+            if dns_rules:
+                rule = dns_rules[0]
+                console.log(f"[green]{rule.info}")
+                status.update(f"{msg_status} [bold red]{rule.on_error}")
+
+                max_retries -= 1
+                sleep(10)
+
+            if max_retries == 0:
+                status.stop()
+                continue_ = Confirm.ask("Continue?")
+                if continue_:
+                    status.start()
+                    max_retries = 5
+                else:
+                    raise typer.Exit()
+
+    """Attach option"""
+    if Confirm.ask(f"Attach ressource to [bold green]{fqdn}"):
+        await attach_resource(account, fqdn, item_hash)
+
+    raise typer.Exit()
+
+
+@app.command()
+@coro
+async def attach(
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    fqdn: str = typer.Argument(..., help=help_strings.CUSTOM_DOMAIN_NAME),
+    item_hash: Optional[str] = typer.Option(None, help=help_strings.CUSTOM_DOMAIN_ITEM_HASH),
+):
+    """Attach resource to a Custom Domain."""
+    account: AccountFromPrivateKey = _load_account(private_key, private_key_file)
+
+    await attach_resource(account, fqdn, item_hash)
+    raise typer.Exit()
+
+@app.command()
+@coro
+async def detach(
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    fqdn: str = typer.Argument(..., help=help_strings.CUSTOM_DOMAIN_NAME)
+):
+    """Unlink Custom Domain."""
+    account: AccountFromPrivateKey = _load_account(private_key, private_key_file)
+
+    await detach_resource(account, fqdn)
+    raise typer.Exit()
+
+
+@app.command()
+@coro
+async def info(
+    private_key: Optional[str] = typer.Option(
+        sdk_settings.PRIVATE_KEY_STRING, help=help_strings.PRIVATE_KEY
+    ),
+    private_key_file: Optional[Path] = typer.Option(
+        sdk_settings.PRIVATE_KEY_FILE, help=help_strings.PRIVATE_KEY_FILE
+    ),
+    fqdn: str = typer.Argument(..., help=help_strings.CUSTOM_DOMAIN_NAME)
```

Comment:
"FQDN" is not a very common abbreviation. Would it make sense to call it (even if wrongly) "URL" and allow people to paste full urls and strip down the unnecessary protocol part?